backend/infrahub/core/migrations/schema/node_attribute_remove.py

```diff
@@ -60,7 +73,8 @@ def render_sub_query_per_rel_type(rel_type: str, rel_def: FieldInfo) -> str:
         query = """
         // Find all the active nodes
         MATCH (node:%(node_kind)s)
-        WHERE exists((node)-[:HAS_ATTRIBUTE]-(:Attribute { name: $attr_name }))
+        WHERE (size($kinds_to_ignore) = 0 OR NOT any(l IN labels(node) WHERE l IN $kinds_to_ignore))
```

Comment:
If `kinds_to_ignore` is empty list, would the second condition be enough and so we can remove `size($kinds_to_ignore) = 0`?